darwin/exporter/formats/mask.py

```diff
@@ -84,6 +84,7 @@ def get_rgb_colours(categories: dt.MaskTypes.CategoryList) -> Tuple[dt.MaskTypes
     dt.MaskTypes.RgbPalette
         A dict of categories and their corresponding RGB palette value.
     """
+    # categories = set(categories)
```

Comment:
So I actually tested something like this, but the problem here is that each time this function gets called, it would reset sorting, and then also mucks with colour mapping. Sets are unordered in python.

Reply:
oh sorry I see it's to remove the comment, not to use a set here